### PR TITLE
[feature](jsonb function)change jsonb_extract_string behavior and doc

### PR DESCRIPTION
**Pull request by xiaokang**
# Proposed changes

Issue Number: close [DSIP-016: Support JSON type](https://cwiki.apache.org/confluence/display/DORIS/DSIP-016%3A+Support+JSON+type?src=contextnavpagetreemode)

## Problem summary

Describe your changes.

1. change jsonb_extract_string behavior: convert to string instead of NULL if the type of json path is not string
2. move jsonb tutorial doc to JSONB data type

## Checklist(Required)

1. Does it affect the original behavior: 
    - [x] Yes
    - [ ] No
    - [ ] I don't know
3. Has unit tests been added:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
5. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
6. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md [hunk 1]
```diff
@@ -35,8 +35,775 @@ under the License.
     2. JSONB format is more efficient. Using jsonb_extract functions on JSONB format is 2-4 times faster than get_json_xx on JSON STRING format.
 
 ### example
-    refer to jsob tutorial.
+A tutorial for JSONB datatype including create table, load data and query.
+
+### create database and table
```
Comment (morningman) [inline docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md:40]: level 4

Comment (morningman) [reply docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md:40]: And pay a attention to the following level

File: docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md [hunk 2]
```diff
@@ -35,8 +35,775 @@ under the License.
...
+|   17 |                            [{"k1":"v41","k2":400},1,"a",3.14] | NULL                    |
+|   18 | {"k1":"v31","k2":300,"a1":[{"k1":"v41","k2":400},1,"a",3.14]} | string                  |
+|   26 |                                          {"k1":"v1","k2":200} | string                  |
++------+---------------------------------------------------------------+-------------------------+
+19 rows in set (0.03 sec)
+
+```
 
 ### keywords
+JSONB, JSON, jsonb_parse, jsonb_parse_error_to_null, jsonb_parse_error_to_value, jsonb_extract, jsonb_extract_isnull, jsonb_extract_bool, jsonb_extract_int, jsonb_extract_bigint, jsonb_extract_double, jsonb_extract_string, jsonb_exists_path, jsonb_type
```
Comment (morningman) [inline docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md:808]: only need `JSONB, JSON`

Other functions has their own docs

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.92 seconds
 load time: 469 seconds
 storage size: 17122675646 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221127094359_clickbench_pr_53624.html

File: docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md (hunk 1 above)
Comment (xiaokang) [reply docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md]: thanks for remind, fixed

File: docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md (hunk 2 above)
Comment (xiaokang) [reply docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md:808]: the examples are useful for other functions and referenced by other docs

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**